docs/modules/sql/pages/mapping-to-maps.adoc

```diff
@@ -41,6 +41,39 @@ OPTIONS (
 )
 ```
 
+If key or value is a complex type, you need to enumerate all required columns in the
+`CREATE MAPPING` statement with `EXTERNAL NAME` modifier. For example, to create a mapping
+for `IMap<Employee, Integer>`:
+
+```java
+public record Employee (
+    int emp_id,
+    int dept_id,
+    String name,
+    String surname,
+    int age
+) implements Serializable {}
+```
+
+```sql
+CREATE MAPPING my_map
+(
+    emp_id INT EXTERNAL NAME "__key.emp_id",
+    dept_id INT EXTERNAL NAME "__key.dept_id",
+    name VARCHAR EXTERNAL NAME "__key.name",
+    surname VARCHAR EXTERNAL NAME "__key.surname",
+    age INT EXTERNAL NAME "__key.age",
+    this INT
+)
+TYPE IMap
+OPTIONS (
+    'keyFormat'='java',
+    'valueFormat'='int'
+)
+```
+
+NOTE: You cannot rename `this` field, it is a technical limitation.
```

Comment:
```suggestion
NOTE: You must not rename the `this` field.
```

Reply:
I suggest removing the statement about it being a technical limitation as that level of detail is not required or useful. If you prefer to keep it, you'll need more information on the limitation.

Cannot means they are not able to - if that is the case, you can leave it as-is; otherwise, we are telling them not to do it, so must not makes more sense.

Reply:
> Cannot means they are not able to - if that is the case

Exactly. It is technically impossible.

Reply:
Thanks. In that case, you are quite right to stick with cannot.